docs/explanation/pebble.rst

```diff
@@ -0,0 +1,85 @@
+.. _pebble_explanation_page:
+
+Pebble
+======
+
+.. important::
+    **Pebble is the default entrypoint for all ROCKs!**
+
+Similar to other well-known process managers such as *supervisord*, *runit*, or
+*s6*, `Pebble`_ is a service manager that enables the seamless orchestration of
+a collection of local service processes as organised set. The main difference
+is that `Pebble`_ has been designed with custom-tailored features that
+significantly enhance the overall container experience, making it the ideal
+candidate for the container's init process (also known as the entrypoint,
+with PID=1).
+
+Multiple processes in a container?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Containers' best practices advocate the separation of concerns and the adoption
+of a single service per container. With the introduction of `Pebble`_ as the
+ROCKs' entrypoint, this principle is elevated to new heights:
+
+  *if multiple processes rely on shared dependencies and are tightly coupled
```

Comment:
The `Pebble`_ not display like you want i think

Reply:
i was talking about this sentence sorry:

```
if multiple processes rely on shared dependencies and are tightly coupled together (i.e. they serve a single purpose and cannot be executed independently), then the best practice entails orchestrating them within the same container, with `Pebble`_ as their manager.
```

Reply:
ah yes, sry. good catch. fixed